src/borg/archiver.py

```diff
@@ -4424,6 +4475,14 @@ def define_borg_mount(parser):
         keep the last N archives under the assumption that you do not create more than one
         backup archive in the same second).
 
+        When using ``--keep-stdio`` borg will produce a listing of all
+        non-checkpoint archives on stdout (similiar to the behaviour of
+        ``borg list``) and expect a json list of archives to keep on stdin.
+        Each element of the list is expected to be an object containing at least
+        one of the keys ``id`` or ``barchive``, as in the output.
+        Only archives given on stdin will be kept.
+        Empty stdin will be treated like ``[]``.
```

Comment:
I see some danger of  people accidentally killing all archives because of some empty input...

Reply:
True; will change...